zpay32/invoice_test.go

```diff
@@ -296,6 +297,20 @@ func TestDecodeEncode(t *testing.T) {
 				}
 			},
 		},
+		{
+			// Invoice with non-ASCII(UTF-8) description
+			encodedInvoice: "lnbc1pvjluezpp5qqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqypqdpquwpc4curk03c9wlrswe78q4eyqc7d8d0np4q0n326hr8v9zprg8gsvezcch06gfaqqhde2aj730yg0durunfhv66t8vpvsktnwvkn5kmsf55vt38s47wz8auzwr2whq4y3l3a24gnjmyze3e4nqq8ynfj5yrdmtgvcuksplvhap8r7j97y0ugfvar7un2zgpplrhq5",
```

Comment:
Where is this invoice coming from? Does not match the one that was added in https://github.com/lightningnetwork/lightning-rfc/pull/338

Reply:
self made, should I replace it with that one instead?

Reply:
Yeah, preferably, to ensure compatability

Reply:
Fixed, also renamed variable to match description in bolt